conode/README.md

```diff
@@ -1,56 +1,612 @@
+address for the leader of critical skipchains should be a virtual address. For
+example, if you have two servers:
+* 10.0.0.2 conode-live, also with secondary address 10.0.0.1
+* 10.0.0.3 conode-standby
+
+You can keep both servers running, and use scp to move the DB file from
+conode-live to conode-standby. Both servers should have the same private.toml
+file, which includes the line `Address = "tcp://10.0.0.1:7770"`
+
+If conode-live is down and unrecoverable, you can add 10.0.0.1 as a secondary
+address to conode-standby and start the conode on it. From this moment on, you
+must be sure that conode-live does not boot, or if it does, that it *does not*
+have the secondary address on it anymore. You could do so by not adding the
+secondary address to boot-time configs and only move it manually.
+
+The address 10.0.0.1 will be in the Roster of any skipchains, and nodes which
+are following that skipchain will still be able to contact the leader, even if
+it is now running on a different underlying server.
+
+Note: The address part of a server identity has name resolution applied to it.
+Thus it would be possible to set the roster of a skipchain to include a server
+identity like "tcp://conode-master.example.com:6979" and then change the
+definition of conode-master.example.com in DNS to change the IP address
+of the master.
+
+## Verifying your server
+
+You can check if the configuration file is correct with:
+
+```bash
+conode -d 3 check ~/.config/conode/public.toml
+```
+
+## Setting up more than one node
+
+You can start multiple nodes on the same server by using one user per node and
+set up the nodes as described above. Be sure to change the port-numbers and
+remember that two ports are used.
+
+## Joining the dedis-cothority
+
+The only existing cothority for the moment is available at
+http://status.dedis.ch. You can send us an email at dedis@epfl.ch to be added to
+this list.
+
+## Compiling your docker file
+
+To create your docker-image and use it, you can create it like this:
+
+```bash
+$ go get github.com/dedis/cothority
+$ cd $(go env GOPATH)/src/github.com/dedis/cothority/conode
+$ make docker
+```
+
+If you use `make docker_run` the first time, a directory called `conode_data`
+will be created and you will be asked for a port - use 7770 or adapt the
+Makefile - and a description of your node. Your public and private key for the
+conode will be stored in `conode_data`. If you run `make docker_run` again, the
+stored configuration will be used.
+
+To stop the docker, simply run `make docker_stop` or kill the docker-container.
+All the configuration is stored in `conode_data`
+
+## Development version with docker
+
+For the latest and greatest version of the conode, you can replace
+`conode:latest` with `conode:dev` and you should get a stable, but changing
+conode. This means, that to use all the functionalities you need to update the
+apps and follow the latest `conode:dev` container regularly.
+
+## Creating Your Cothority
```

Comment:
this section seems to be out-of-place, it's in between sub-sections about docker. maybe combine it with "Setting up more than one node" or "Verifying your server"? you can tell the user to create the combined roster.toml and then the `conode check` command should check all the conodes.